ci-operator/step-registry/servicemesh/kiali/OWNERS

```diff
@@ -0,0 +1 @@
+../OWNERS
```

Comment:
Does this file to need to be filled out? As well as a few other OWNERS files. Or does this allow you to point to a different OWNERS file?